### ISSUE TITLE
UX/UI: Achievements contrast check

### ISSUE THREAD

**Issue by SaskiaSteyn** [SaskiaSteyn/tasktails#283]
Currently the achievements that are not unlocked, fail contrast check. Up the contrast on the text on the background, and make sure it still looks locked. Then ensure it isn't disabled so that a screen reader can still read it.
